acceptance/login_test.go

```diff
@@ -177,3 +227,38 @@ var _ = Describe("Login", func() {
 		Expect(outLines[1]).To(ContainSubstring(randomPort + `: connect: connection refused`))
 	})
 })
+
+// getAuthorizationCode will read the output from reader looking for the authentication url where to login,
+// Then it will try to login, get the authentication code and write it to the stdin
+func getAuthorizationCode(reader io.Reader, stdinPipe io.Writer) {
+	defer GinkgoRecover()
+
+	// wait a second for the command to execute
+	time.Sleep(time.Second)
```

Comment:
maybe read from reader until the output we expect appears? I'm not sure if we can immediately start reading and how the final code will look but let's try to avoid magic sleeps

Reply:
Yes, it makes sense. Let me try that. :slightly_smiling_face: